docs/DeveloperGuide.md

```diff
@@ -925,14 +926,40 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
+### Adding a contact
+
+1. Adding a contact with all fields
+
+    1. Prerequisites: No existing contact with phone number 98989898 and existing contact with phone number 32323232.
+
+    1. Test case: `add n/James p/98989898 e/james@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney b/04071999`<br>
+       Expected: Contact with correct details added to contact list. Contact's birthday reminder message added to birthday reminder list in correct position. Details of the added contact shown in the status message.
```

Comment:
I think birthday reminder message is a bit weird. Maybe Contact's birthday is added is better?